SCClassLibrary/Common/Control/Server.sc

```diff
@@ -725,13 +773,9 @@ Server {
 		^Buffer.cachedBufferAt(this, bufnum)
 	}
 
-	// defaultGroups for all clients on this server:
-
-	allClientIDs { ^(0..options.maxLogins-1) }
```

Comment:
should this be properly deprecated rather than hard removal?

Reply:
this hasn't been part of any release, it was added in a recent PR

Reply:
understood